.buildkite/release-automation/verify-linux-wheels.sh

```diff
@@ -4,9 +4,9 @@ set -euo pipefail
 
 set -x
 
-export PYTHON_VERSION="${PYTHON_VERSION:-3.8}"
-export RAY_VERSION="${RAY_VERSION:-2.9.1}"
-export RAY_HASH="${RAY_HASH:-cfbf98c315cfb2710c56039a3c96477d196de049}"
+export PYTHON_VERSION="${PYTHON_VERSION}"
+export RAY_VERSION="${RAY_VERSION}"
+export RAY_HASH="${RAY_HASH}"
```

Comment:
maybe throw/fail if these RAY_VERSION/RAY_HASH value is empty

Reply:
done!

Reply:
If RAY_VERSION, etc. are already part of the global environment, do we need to export it again?

Reply:
we do, otherwise it won't be visible in sub commands.